### PR TITLE
added github url in body and resolved some issues

### PR DESCRIPTION
**Pull request by omkarkhatavkar**
### PR Objective 
1. Resolve the common issues, do not add username in PR_create and Issue_create.
2. Add JiraUsername in only PR_Review and add them in Search Criteria 
3. Modified #pr to #pr_create to make sure that duplicate tasks should not be created.
4. Added Github url in details   (https://github.com/omkarkhatavkar/jirasync/issues/8)

### CODE REVIEW

**Comment by omkarkhatavkar** [comment]
### Execution :

image

### Jira Result 

image

File: wrappers/workflow_wrapper.py [hunk 1]
```diff
@@ -29,11 +31,12 @@ def create_pull_request_in_current_sprint(issue, jira, issue_text, assignee=None
...
             params = {
-                'summary': "{}#{} -{}".format(issue_text, assignee, issue.title),
-                'details': "{}".format(issue.data['body']),
+                'summary': "{} -{}".format(issue_text, issue.title),
+                'details': "Source : {}\n{}".format(issue.data['html_url'].encode(),
+                                                    issue.data['body']),
                 'component': 'Automation',
                 'labels': ['Automation'],
-                'sprint': jira.current_sprint_name.encode(),
+                'sprint': 'backlog',
```
Comment (omkarkhatavkar) [inline wrappers/workflow_wrapper.py]: while testing I accidentally it was changed to the backlog. This will not work in local docker Jira instance, as we don't have sprint configured

@rochacbruno we can keep it backlog, for the time being, then we can change it to current sprint

Comment (rochacbruno) [reply wrappers/workflow_wrapper.py]: or we can build the sprint earlier in a try block:

```python
if issue_text is not None:
    try:
        sprint =  jira.current_sprint_name.encode()
    except WhateverExceptionitRaises:
       sprint = 'backlog'
   ...
   params = {
       ...
       'sprint': sprint,
       ...
   }
```

File: plugins/jira_workflow.py [hunk 2]
```diff
@@ -34,7 +34,7 @@ def start_issue_workflow(github_issues, jira, assignee=None):
 def start_create_pull_requests_workflow(github_issues, jira, assignee=None):
     # Create PR task if those are not created
     for issue in github_issues:
-        pr_text = str('pr#' + issue.project + '#' + issue.id)
+        pr_text = str('pr_create#' + issue.project + '#' + issue.id)
```
Comment (rochacbruno) [inline plugins/jira_workflow.py]: it is a good place to us `format`

```python
pr_text = "pr_create#{issue.project}#{issue.id}".format(issue=issue)
```

Not a blocker, but format is more readable and already converts to `str`

File: wrappers/workflow_wrapper.py (hunk 1 above)
Comment (omkarkhatavkar) [reply wrappers/workflow_wrapper.py]: that's good, let me rephrase it.

File: plugins/jira_workflow.py (hunk 2 above)
Comment (omkarkhatavkar) [reply plugins/jira_workflow.py]: OK
